experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_experiment_clone_serializer.py

```diff
@@ -119,3 +119,24 @@ def test_locales_not_clone_for_mobile_application(self):
         self.assertTrue(serializer.is_valid())
         child = serializer.save()
         self.assertEqual(list(child.locales.all()), [])
+
+    def test_release_date_not_clone_for_mobile_application(self):
+        mobile_application = NimbusExperiment.Application.FENIX
```

Comment:
I wouldn't bother pulling this out into a separate variable since it is only used once.